src/Compilers/VisualBasic/Portable/Scanner/KeywordTable.vb

```diff
@@ -297,7 +298,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax
 
             Dim kind As SyntaxKind
             If Not s_keywords.TryGetValue(tokenName, kind) Then
-                kind = SyntaxKind.IdentifierToken
+                ' s_keywords includes preprocessor keywords, but only one of
+                ' "r" and "reference".  GetProcessorKeywordKind supports both.
```

Comment:
Also, are we also supporting \#reference? it is not clear why "refernce" is important. Please clarify the comment.